runtests.py

```diff
@@ -241,9 +242,12 @@ def main(argv):
             bench_args.extend(['--bench', a])
 
         if not args.bench_compare:
-            cmd = [os.path.join(ROOT_DIR, 'benchmarks', 'run.py'),
-                   'run', '-n', '-e', '--python=same'] + bench_args
-            os.execv(sys.executable, [sys.executable] + cmd)
+            import scipy
+            print("Running benchmarks for Scipy version %s at %s"
+                  % (scipy.__version__, scipy.__file__))
+            cmd = ['asv', 'run', '--dry-run', '--show-stderr',
+                   '--python=same'] + bench_args
+            run_asv(cmd)
```

Comment:
`run_asv` returns the result of `subprocess.call` or the literal `1`, but we ignore the return value here. Perhaps we should do `sys.exit(run_asv(cmd))` ?

Reply:
Good catch, fixed.